src/tests/posix/pthread/pthread_kill_test.c

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file
+ * @brief test for pthread_kill
+ *
+ * @author  XU Chenglin
+ * @date    09/03/2020
+ */
+
+#include <pthread.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <signal.h>
+#include <embox/test.h>
+
+
+EMBOX_TEST_SUITE("posix/pthread_kill");
+
+
+void sighand(int signo)
+{
+    return;
```

Comment:
empty handler? May be add some functionality in it?

Reply:
May I add test_assert_not_zero(signo) here? Do you have any other suggestion? 
Thank you so much!